setup.cfg

```diff
@@ -1,12 +1,12 @@
 [flake8]
-exclude = migrations
-ignore = F405,W503,E731
+exclude = .git,__pycache__,docs,build,dist,site,migrations,node_modules
+ignore = F405,E501,W503,E731,E203
 max-complexity = 10
 max-line-length=119
 
 [isort]
+profile = black
 line_length = 79
-multi_line_output = 4
```

Comment:
Removed this as this seemed to be the cause of the weird formatting for multi-line imports